### ISSUE TITLE
COVID-19 disease progression & category

### ISSUE THREAD

**Issue by johnl8888** [National-COVID-Cohort-Collaborative/Phenotype_Data_Acquisition#8]
The current 4-tier proposal may need additional layer to capture the multi-faceted nature of this COVID-19 disease; and the lagged codings in EHR may hinge how we collect data for downstream research, we need think a bit out of the box for this brand new phenotype. 

Based on the known observation thus far, the disease displays spectrum of severity, for example, Severity classification according to Australian guidelines for the clinical care of people with COVID-19 (v2) (https://app.magicapp.org/app#/guideline/4179) - mild/moderate/severe/critical. 

Many mild patients may not have symptoms - those patients will not likely be recorded or coded in EHR, but may be rediscovered by antibody tests and recorded in EHR later or different systems. Although we'll not anticipate those severity spectrum being assigned different ICD10 codes at this early stage, however I believe introduce this phenotype spectrum plus the proposed tier concept will prepare us to handle both known and unknowns better down the line.

**Comment by remurphymd** [comment]
I agree with this concept and I like the Australian model for tiering. However, we might be limited by not have details such as respiratory rates or oxygen saturation in all data sets
